src/som/interpreter/LexicalScope.java

```diff
@@ -267,8 +301,7 @@ public void setMethod(final Method method) {
       CompilerAsserts.neverPartOfCompilation("LexicalContext.sOM()");
       // might be reset when doing inlining/embedded, but should always
       // refer to the same method
-      assert this.method == null ||
-          this.method.getSourceSection() == method.getSourceSection();
+      //assert this.method == null || this.method.getSourceSection() == method.getSourceSection();
```

Comment:
Neither condition is valid anymore?
Why not remove the line then?
Though, i think the second might hold?